src/main/java/com/bootcamp/demo/controller/LoginController.java

```diff
@@ -0,0 +1,27 @@
+package com.bootcamp.demo.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.ui.ModelMap;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
```

Comment:
Remove unnecessary import. 

**Important for all team members**: always optimise the imports before committing the files. You can do this by opening the class in Intellij, and accessing from the menu Code -> Optimize imports (or directly using the keyboard shortcut).